gst/tensor_filter/tensor_filter.c

```diff
@@ -530,6 +572,46 @@ gst_tensor_filter_set_property (GObject * object, guint prop_id,
         g_strfreev (str_types);
       }
       break;
+    case PROP_INPUTNAME:
+      g_assert (!prop->input_configured && value);
+      /* Once configures, it cannot be changed in runtime */
+      {
+        int i;
```

Comment:
For tensorflow-lite, inputname may be a null string "", not null. Then, you will give GST_ERROR.

Think about how youw are going to handle property inconsistency...